tests/testthat/test-use_crate.R

```diff
@@ -0,0 +1,28 @@
+test_that("use_crate() adds dependency to package or workspace", {
```

Comment:
Could you please add a case for unhappy path? E.g. both version and git?

Reply:
i also added the "rextendr_error" class to the `cli_abort()` call